builds/misc/images-release.yaml

```diff
@@ -0,0 +1,602 @@
+        parameters: 
+          imageName: azureiotedge-agent
+          name: "Edge Agent"
+          project: Microsoft.Azure.Devices.Edge.Agent.Service
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-hub
+          name: "Edge Hub"
+          project: Microsoft.Azure.Devices.Edge.Hub.Service
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-simulated-temperature-sensor
+          name: "Temperature Sensor"
+          project: SimulatedTemperatureSensor
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics
+        displayName: 'Build Image - azureiotedge-diagnostics - amd64'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch armv7l &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch armv7l
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM32'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch aarch64 &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch aarch64
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM64'
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Artifacts to VSTS'
+        inputs:
+          PathtoPublish: '$(Build.BinariesDirectory)/publish'
+          ArtifactName: 'publish-linux'
+################################################################################
+  - job: windows
+################################################################################
+    displayName: Windows
+    pool:
+      name: Azure-IoT-Edge-Core
+      demands: 
+        - Build-Image -equals true
+        - win-rs5
+        - DotNetFramework
+    variables:
+      build.registry.address: edgebuilds.azurecr.io
+      build.registry.user: EdgeBuilds
+      registry.address: edgerelease.azurecr.io
+      registry.user: EdgeRelease
+    steps: 
+      - script: |
+            docker login $(build.registry.address) -u $(build.registry.user) -p $(build.registry.password)
+            docker login $(registry.address) -u $(registry.user) -p $(registry.password)
+        displayName: 'docker login'
+      - template: ../templates/install-dotnet3.yaml
```

Comment:
do you miss thiu line - template: ../templates/install-dotnet2.yaml?  we are using MS hosted agent.  
- template: ../templates/install-dotnet2.yaml and - template: ../templates/install-dotnet3.yaml are not required.

Reply:
is missing - template: ../templates/dotnet3-globaljson.yaml here?

Reply:
windows doesn't need this logic

Reply:
it is curious, do you know why it doesn't need on Windows?